src/Contrib/Context/Swoole/composer.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "open-telemetry/context-swoole",
+  "description": "Async Swoole context implementation for OpenTelemetry PHP.",
+  "keywords": ["opentelemetry", "otel", "context", "swoole", "coroutine"],
+  "type": "library",
+  "license": "Apache-2.0",
+  "authors": [
+    {
+      "name": "opentelemetry-php contributors",
+      "homepage": "https://github.com/open-telemetry/opentelemetry-php/graphs/contributors"
+    }
+  ],
+  "require": {
+    "php": "^7.4 || ^8.0",
+    "open-telemetry/context": "self.version",
+    "ext-openswoole": "^4"
```

Comment:
@Nevay - this was an educated guess as to the swoole requirements

Reply:
I don't think we should add this constraint, the code should be compatible with `ext-swoole` and `ext-openswoole`.

Reply:
removed, in favour of mentioning both swoole variants in description and tags.